sky/cli.py

```diff
@@ -995,13 +995,14 @@ def cli():
           'running ``sky launch -d ...`` and then ``sky autostop -i <minutes>``'
           '. If not set, the cluster will not be auto-stopped.'))
 @click.option(
-    '--terminate',
+    '--down',
     default=False,
     is_flag=True,
     required=False,
-    help=(
-        'Terminate the cluster after execution. If --idle-minutes-to-autostop '
-        'is set, the cluster will be torn down after the idle time.'),
+    help=
+    ('Tear down the cluster after execution (successfully or abnormally). If '
+     '--idle-minutes-to-autostop is set, the cluster will be torn down after '
+     'the idle time.'),
```

Comment:
Nit: 
```
    ('Tear down the cluster after execution finishes (successfully or abnormally). If '
     '--idle-minutes-to-autostop is also set, the cluster will be torn down after '
     'the idle time, rather than immediately after execution finishes.'),
```
Just to clarify the second part a bit.